packages/generator-sketch/src/config.js

```diff
@@ -344,35 +424,38 @@ module.exports = {
         var helperMessage = findLayer(symbol, (s) =>
           /^div#helper-message/.test(s.name)
         );
-        if (helperMessage) helperMessage.resizingConstraint = 34;
-        if (!helperMessage && info) info.resizingConstraint = 34;
-        if (!helperMessage && count) count.resizingConstraint = 34;
+        if (helperMessage)
+          helperMessage.resizingConstraint = BOTTOM_LEFT_RIGHT_FIXED_HEIGHT;
```

Comment:
do you think it's better to use curly brackets for the conditions?

Reply:
depends, but i agree in this case
(the whole block should be eventually replaced by a switch or a map anyways  🙃)

Reply:
so do you want to refactor this then to make it less scary? :)

Reply:
yep, but in another PR
this need a little more declustering 🙃